src/Card.css

```diff
@@ -1,6 +1,40 @@
 li {
   border: 2px solid white;
-  height: 300px;
-  width: 100px;
+  height: 400px;
+  width: 300px;
   margin: 0 3px 0 3px;
```

Comment:
It looks like there may be some duplication in the css with this margin that we could clean up a bit.